src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+@import "../utils/variables";
+@import "../utils/mixins";
+
+.card {
+  &:nth-child(-n + 4) {
+    margin-bottom: $gap;
+  }
+
+  @include cardStyle();
+
+  & > * {
+    display: flex;
+  }
+
+  &:not(:nth-child(4n)) {
+    margin-right: $gap;
+  }
+
+  &__image {
+    max-width: 160px;
+    height: 134px;
+  }
+
+  &__image--possition {
+    margin: 32px 19px 40px;
+  }
+
+  &__model--possition {
+    margin: 0 16px 4px;
+  }
```

Comment:
Don not use margin for positioning. Remove all modifiers you add for this

Reply:
what we need to use instead of these? paddings?